src/control/cmd/daos_server/README.md

```diff
@@ -0,0 +1,67 @@
+# DAOS Server
+
+The DAOS Server can be run either as a foreground process to issue hardware
+provisioning commands or as a background process (binary name followed by the
+"start" subcommand) when it will listen for instructions from the [management
```

Comment:
Could add a link to existing docs or add a bit more detail directly here regarding the hardware provisioning element.